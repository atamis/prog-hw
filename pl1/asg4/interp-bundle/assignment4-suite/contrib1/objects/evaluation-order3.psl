# Using bracket lookup the object should be evaluated before the field name
({Second: print("First\n")})[print("Second")];
print("\n");
({Second: print("First\n")})[print("Second")] = print("\nThird");
"";
