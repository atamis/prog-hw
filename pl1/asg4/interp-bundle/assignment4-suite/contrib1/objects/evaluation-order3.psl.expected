First
Second
First
Second
Third
